cheat.php

```diff
@@ -734,7 +734,8 @@ function Msg( $Message, $EOL = PHP_EOL, $printf = [] )
 
 	if( !empty( $printf ) )
 	{
-		printf( $Message, ...$printf );
+		array_unshift($printf, $Message);
```

Comment:
Can you add spaces to match the file style

Reply:
Fixed. Paid attention to the tabs, but not to that